src/components/IstioConfigPreview/IstioConfigPreview.tsx

```diff
@@ -20,62 +27,67 @@ import { EditResources } from './EditResources';
 import { cloneDeep } from 'lodash';
 import _ from 'lodash';
 
-type PolicyItem = AuthorizationPolicy | Sidecar;
+export type PolicyItem =
```

Comment:
"PolicyItem" is confusing. It's an Istio config resource, so "IstioConfigItem"/"IstioConfigResource" may help to not get lost on these layers.